hieradata_aws/class/staging/db_admin.yaml

```diff
@@ -29,3 +29,33 @@ govuk_env_sync::tasks:
     temppath: "/tmp"
     url: "govuk-staging-database-backups"
     path: "mysql"
+  "pull_pg_content_publisher_production_daily":
+    hour: "0"
+    minute: "12"
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "content_publisher_production"
+    temppath: "/tmp/content_publisher_production"
```

Comment:
Should we be consistent about `temppath` locations? They seem to vary between `/home/govuk-backup/...` and `/tmp`.

Reply:
We can optimize this inconsistency when we've tested that all these jobs work.